controllers/suite_test.go

```diff
@@ -137,6 +138,10 @@ var _ = BeforeSuite(func() {
 	Expect(err).NotTo(HaveOccurred())
 	err = os.Setenv("METRICS_EXPORTER_KUBE_RBAC_PROXY_IMAGE", "mock-image")
 	Expect(err).NotTo(HaveOccurred())
+	err = os.Setenv("METRICS_EXPORTER_PROMETHEUS_OPERATOR_SERVICE_ACCOUNT", "k8s-prometheus")
```

Comment:
do you need the new variable here?

Reply:
no, I set it on the fly here:
https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/687/files#diff-35c949584634dae6fb47d82556f92d648986892e843a64d1eb9bd294375a61dcR370

So I can test both values